<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageFilm</class>
 <widget class="QWizardPage" name="WizardPageFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Page 2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QListWidget" name="w_acteur"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Durée</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="w_type"/>
   </item>
   <item row="2" column="1">
    <widget class="QTimeEdit" name="w_duree"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="w_realisateur"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Réalisateur</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Scenariste</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="w_scenariste"/>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="w_ajouter">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="w_supprimer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>w_realisateur</tabstop>
  <tabstop>w_scenariste</tabstop>
  <tabstop>w_duree</tabstop>
  <tabstop>w_type</tabstop>
  <tabstop>w_ajouter</tabstop>
  <tabstop>w_supprimer</tabstop>
  <tabstop>w_acteur</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
